plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -82,10 +102,15 @@ export const Reader = ({ entityId, onReady }: Props) => {
           --md-code-bg-color: ${theme.palette.background.paper};
         }
         .md-main__inner { margin-top: 0; }
-        .md-sidebar { top: 0; width: 20rem; }
+        .md-sidebar {  position: fixed; bottom: 100px; width: 20rem; }
```

Comment:
The current experience on smaller viewports is not great, but with these changes, it kinda becomes unusable.  Is there anything we can do to improve?

![techdocs-smaller-layouts](https://user-images.githubusercontent.com/3496491/112190694-8a6c2380-8c05-11eb-8c33-f2b5070646b9.gif)

Reply:
https://user-images.githubusercontent.com/14866712/112248587-1ecf9800-8c91-11eb-8f8f-84fb5ac98a11.mp4

At narrow screen width the sidebars disappear and the footer links go back into the flow, at the bottom of the page. This is similar to Mkdocs's default handling of narrow screen width.

Suggest better handling of the left sidebar (move into a button and dialog perhaps) can be done as a separate issue.

Reply:
Seems some leftover css from Mkdocs' way of handling small screens might be reusable for us. I'll give it a try first.

Reply:
https://user-images.githubusercontent.com/14866712/112268458-f73cf780-8cb1-11eb-91b4-9fae02e4bbce.mov

Modified some of the CSS from MkDocs' original styling to allow rendering a compact version of the left sidebar in small views. Footer nav links are placed back in the flow at the bottom in narrow screen mode.

Reply:
Nice!